nubis/bin/nubis-deploy

```diff
@@ -29,7 +29,28 @@ ansible-playbook -i localhost, \
   -e template=$CF_TEMPLATE \
   -e @$VARIABLES
 
+export CONSUL_ENDPOINT
+CONSUL_ENDPOINT="public.consul.$environment.$region.$NUBIS_ACCOUNT.$NUBIS_DOMAIN"
```

Comment:
Actually, not really, it doesn't matter. *export* merely marks a shell variable to be exported to sub-processes.